README.md

```diff
@@ -34,6 +34,7 @@ The [`my-app/.templates`](.templates) dir contains some templates you can use in
 
 - `<APP_NAME>`: the application name
 - `<APP_SHORT_DESCRIPTION>`: a quick app description
+- `<APP_CATEGORY>`: the app category (used in the cozy-bar apps navigation)
```

Comment:
Since we juste have `cozy` and `partners` and that could change soon I didn't add it for now. Do you think it's interesting to mention them at this stage?